decidim-core/app/controllers/decidim/group_email_confirmations_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Decidim
+  # The controller to manage email confirmations for user groups.
+  class GroupEmailConfirmationsController < Decidim::ApplicationController
```

Comment:
Should we keep the naming and rename this to `UserGroupEmailConfirmationsController`?

Reply:
We have a lot of controllers called `Group**Controllers`